google-cloud-bigtable-stats/src/main/java/com/google/cloud/bigtable/stats/BigtableStackdriverStatsExporter.java

```diff
@@ -76,9 +84,15 @@ public static void register(Credentials credentials) throws IOException {
           instance == null, "Bigtable Stackdriver stats exporter is already created");
       // Default timeout for creating a client is 1 minute
       MetricServiceClient client = createMetricServiceClient(credentials, Duration.create(60L, 0));
-      MonitoredResource resourceType =
-          MonitoredResource.newBuilder().setType(RESOURCE_TYPE).build();
-      instance = new BigtableStackdriverStatsExporter(client, EXPORT_INTERVAL, resourceType);
+      MonitoredResource bigtableResourceType =
+          MonitoredResource.newBuilder().setType(BIGTABLE_RESOURCE_TYPE).build();
+      MonitoredResource gceResourceType =
+          MonitoredResource.newBuilder().setType(GCE_RESOURCE_TYPE).build();
+      MonitoredResource gkeResourceType =
+          MonitoredResource.newBuilder().setType(GKE_RESOURCE_TYPE).build();
+      instance =
+          new BigtableStackdriverStatsExporter(
```

Comment:
I don't think we need to pass in the monitored resource type from here. Can we create these monitored Resource in ExporterUtil#convertTimeSeries?